cgi-bin/dashboard-aii.cgi

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/perl -T
+# ${license-info}
+# ${developer-info
+# ${author-info}
+# ${build-info}
+
+use Config::Simple;
+use JSON::XS;
+use Switch;
+use CGI;
+
+# Configuration file to boot from HD. Following pxelinux convention
+# it must be called default so if the <IpAddressInHex> link
+# is missing the node will boot from the hard disk
+my $boothd = "default";
+
+# Quattor profiles
+my $web_root = "/var/www/html";
+my $profiles_path = "profiles";
+my $profiles_dir = $web_root.'/'.$profiles_path;
+
+# Read AII configuration
+$cfg = new Config::Simple();
+$cfg->read('/etc/aii/aii-shellfe.conf');
+
+# PXE Linux directory
+my $pxelinux_dir = $cfg->param('nbpdir');
+
+# Profile prefix
+my $profile_prefix = $cfg->param('profile_prefix') eq 0 ? '' : $cfg->param('profile_prefix');
```

Comment:
I have in my aii-shellfe.conf : profile_prefix = 0. I don't know how this value is threated by aii itself. Is it required by aii and what type is used (I don't understand how 0 is known as empty for AII) ?